scripts/example.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
i think the name should be something more descriptive than example.json. exampleDaoData.json? also maybe this should go in the test directory?